### PR TITLE
Default search engine functional UI changes

### PR DESCRIPTION
**Pull request by imptrx**
## Changes
- Change flow for how users select their default search engine. Gives user ability to use immediate dropdown rather than go through settings
- https://github.com/brave/brave-browser/issues/1548
- Quick View:
search_defaults

## Test plan
- Ensure dropdown works
- Ensure 'Confirm' button still links to next page of on-boarding flow
- Ensure responsiveness on smaller screens

##### Link / storybook path to visual changes
- http://localhost:9091/?path=/story/feature-components-welcome--page
- Third step of the onbording flow

## Integration
- [ ] Does this contain changes to src/components or src/
  - [ ] Will you publish to npm immediately after this PR, or wait until sometime in the future?
  - [ ] Incompatible API change to something existing _(major version increase)_
  - [ ] Adding new backwards-compatible functionality? _(minor version increase)_
  - [ ] Fixing a bug backwards-compatibly? _(patch version increase)_
  
- [x] Does this contain changes to src/features for brave-core?
Only change here that will be directly impacting brave-core will the the SVG update
  - [ ] Are there non backwards-compatible changes required for brave-core? **Do not merge until brave-core PR is approvable.** Link to brave-core PR:
https://github.com/brave/brave-core/pull/2479
  - [ ] Will you create brave-core PR to update to this commit after it is merged?
  - [ ] Wants uplift to brave-core feature branch?
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit

### CODE REVIEW

File: stories/features/welcome/page/screens/searchBox.tsx [hunk 1]
```diff
@@ -6,7 +6,8 @@
 import * as React from 'react'
 
 // Feature-specific components
-import { Content, Title, Paragraph } from '../../../../../src/features/welcome/'
+import { Content, Title, Paragraph, SelectGrid } from '../../../../../src/features/welcome/'
+import { SelectBox } from '../../../../../src/features/shields'
```
Comment (imptrx) [inline stories/features/welcome/page/screens/searchBox.tsx]: Tentatively using selectBox from shields - will finalize component structure after migration of features/stories to `brave-core`

File: stories/features/welcome/page/fakeLocale.ts [hunk 2]
```diff
@@ -13,8 +13,11 @@ const locale = {
   import: 'Import',
   imported: 'Imported',
   // third screen
-  setDefaultSearchEngine: 'Change your search engine',
-  chooseSearchEngine: 'Choose the search engine you would like to use by default when searching the web from the address bar.',
+  setDefaultSearchEngine: 'Set default search engine',
+  chooseSearchEngine: 'Choose the search engine you would like to use by default when searching the web from the URL bar.',
```
Comment (imptrx) [inline stories/features/welcome/page/fakeLocale.ts]: Using string taken from linked ticket from brave-browser, need to double check which is more current

**Comment by rossmoody** [comment]
Got some updates here. 

### Headline 
`Set default search engine`

### Body copy
`Choose the search engine you would like to use by default when searching the web from the address bar. For a more private search experience, try DuckDuckGo.`

### Button
`Set default`

The button should remain disabled until a selection is made in the dropdown. If the button is hovered while disabled, a tooltip is presented. If a selection is made, the screen should auto progress on button press. 

### Tooltip
`Select a new default search engine or Google will be used by default.`

### Dropdown
I believe there are regionally specific considerations for this but DuckDuckGo and Qwant should be listed first in the order of any available options. 

### SVG
[This can replace the current search XML.](https://d.pr/n/893Ubc) 

SearchEngineNotes

**Comment by rossmoody** [comment]
Also, hi @imptrx! Glad to have you aboard.

**Pull request merged**